src/CMakeLists.txt

```diff
@@ -86,9 +86,11 @@ set(keepassx_SOURCES
     gui/FileDialog.cpp
     gui/IconModels.cpp
     gui/KeePass1OpenWidget.cpp
+    gui/kmessagewidget.cpp
```

Comment:
Shouldn't the filename also be PascalCase, even though this is the original filename?